workloads/kube-burner-ocp-wrapper/run.sh

```diff
@@ -7,7 +7,7 @@ LOG_LEVEL=${LOG_LEVEL:-info}
 if [ "$KUBE_BURNER_VERSION" = "default" ]; then
     unset KUBE_BURNER_VERSION
 fi
-KUBE_BURNER_VERSION=${KUBE_BURNER_VERSION:-1.2.0}
+KUBE_BURNER_VERSION=${KUBE_BURNER_VERSION:-1.2.5}
```

Comment:
It wasn't mentioned in the PR message you were bumping the version of kube-burner?

Reply:
I am still running tests using this PR. will let you know once it is ready to go.